.github/workflows/release.yml

```diff
@@ -53,5 +53,5 @@ jobs:
           commit: "release: version packages"
           publish: yarn publish
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GH_TOKEN }}
```

Comment:
I'm switching this to a PAT so that it can trigger other Github Actions (like the docs auto-generator)